script.video.randomtv/addon.py

```diff
@@ -167,11 +177,33 @@ def onPlayBackStopped(self):
 
 	# Start Player
 	player.play(item=myPlaylist)
-	xbmc.sleep(100)
 #
 
 
-while (not xbmc.Monitor().abortRequested()):
+while (not xbmc.Monitor().waitForAbort(0.1)):
```

Comment:
Is it really necessary to do it every 0.1? Might slow down some low powered devices. Higher is better here as long as your addons responsiveness doesn't change from it.